Add information that Kuryr doesn't support unidling (#41097)
openshift/openshift-docs: modules/idle-unidling-applications.adoc

Automatic unidling by a router is currently only supported by the default
HAProxy router.
====
[NOTE]
====

> If Kuryr-Kubernetes is configured as an SDN, automatic unidling by services is not supported.

Reviewer: Small change for active voice per ISG
```suggestion
Services do not support automatic unidling if you configure Kuryr-Kubernetes as an SDN.
```

Automatic unidling by a router is currently only supported by the default
HAProxy router.
====
[NOTE]
====

> Automatic unidling by a service is not supported when Kuryr-Kubernetes is

Reviewer 2: "unidling" is definitely the correct jargon to use here? Would love if it weren't, but if it is, it is what it is. :) @dulek

Author: It seems so, judging from the rest of the module I'm modifying. The note I'm adding is analogous to the one above.

> Automatic unidling by a service is not supported when Kuryr-Kubernetes is
> configured as SDN.

Reviewer 2: ```suggestion
If Kuryr-Kubernetes is configured as an SDN, automatic unidling by services is not supported.
```

Does this work?

Author: Sure!
